sites/public/styles/footers.scss

```diff
@@ -4,8 +4,9 @@ footer.site-footer > .footer-row:first-of-type {
 }
 
 .footer-sections {
-  padding: 2rem 0;
-  max-width: 1024px;
+  @apply py-8;
```

Comment:
We're moving off of tailwind, do you mind adding these as vanilla css classes since they're new?

Reply:
@emilyjablonski Just updated!

Reply:
I believe the tokens are available already…just need to swap out any remaining `@apply` or hardcoded values (aka `var(--bloom-s2)` rather than `0.5rem`)